.github/workflows/test.yml

```diff
@@ -332,6 +332,35 @@ jobs:
       - name: Integration Tests
         run: turbo run test --filter=turborepo-tests-integration --color --env-mode=strict --token=${{ secrets.TURBO_TOKEN }} --team=${{ vars.TURBO_TEAM }}
 
+  turborepo_integration_rust:
```

Comment:
Can this be matrixed into the job above? It's 90% the same. An `if: {{matrix.codepath == "rust"}}` shuold do the trick to change the command that runs?

Reply:
We don't want this to be blocking, so I think a second job is best

Reply:
Why not make it blocking? Seems like we're close to the end and should make it blocking at some point